app/javascript/src/components/SceneEditor.tsx

```diff
@@ -2,19 +2,40 @@ import * as React from 'react'
 
 import 'redactor/redactor.css'
 import 'redactor/redactor'
+
+import './ReactVoiceRecorder.css'
 import './SceneEditor.css'
 
 import { get, set } from 'lodash'
 import { DefaultNodeModel } from 'storm-react-diagrams'
+import ReactAudioPlayer from 'react-audio-player'
 import { StateConsumer, ApplicationState } from '../Store'
 import ChoiceEditor from './ChoiceEditor'
 import SettingsEditor from './SettingsEditor'
-import { savePhoto, removePhoto } from '../persistance'
-
+import { savePhoto, removePhoto, saveAudio, removeAudio } from '../persistance'
 
 declare function $R(el: HTMLElement, options: any): void
 declare function $R(el: HTMLElement, fun: string, arg: string): void
 
+const ReactVoiceRecorder = require('react-voice-recorder')
```

Comment:
Any reason this is using the commonjs require instead of ESM import syntax?

Reply:
I could not figure out why, but I could not get the the import to work and stack overflow just suggested using require instead so that is what I did.